.github/workflows/test.yaml

```
@@ -44,4 +29,4 @@ jobs:
         timeout-minutes: 20
         run: |
           PYTHONPATH=$PWD/gnomad_methods:$PWD/seqr-loading-pipelines \
-          python -m pytest test/test_${{ matrix.suffix }}.py
+          pytest test
```

With `pytest-xdist` in the dev requirements we can parallelise out all these tests per core. It looks like the super long running large_cohort workflow test is still a bottleneck at the moment, but if/when that's trimmed down the execution times will drop

```suggestion
          pytest -n auto test
```

---

running it locally the parallel version actually takes a few seconds longer 😓 the bottlenecking around that one test runtime + the few seconds overhead setting up the parallel runs + low overall number of tests. Ignore this!

---

sadface